data/json/items/tool/entry_tools.json

```diff
@@ -119,5 +119,26 @@
     "color": "light_gray",
     "qualities": [ [ "LOCKPICK", 40 ] ],
     "flags": [ "PSEUDO", "PERFECT_LOCKPICK" ]
+  },
+  {
+    "id": "heavy_crowbar",
+    "type": "TOOL",
+    "name": { "str": "heavy crowbar" },
+    "looks_like": "crowbar",
+    "description": "A 5 foot long construction prybar.  Essentially a long solid steel rod with a hardened chisel tip.",
+    "weight": "8 kg",
+    "to_hit": 1,
```

Comment:
Please use this format: https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_INFO.md#to-hit-object

Reply:
I changed the format. It looks right but the linting website is now giving me a syntax error.